src/getConsentFor.ts

```diff
@@ -44,6 +44,15 @@ export const getConsentFor = (
 		return !consent.ccpa.doNotSell;
 	}
 
+	if (consent.aus) {
+		if (typeof consent.aus.rejectedVendors === 'undefined') return true;
+		const rejected = consent.aus.rejectedVendors.filter(
+			// eslint-disable-next-line no-underscore-dangle
+			(rejectedVendor) => rejectedVendor._id === sourcepointId,
```

Comment:
```suggestion
			(rejectedVendor) => rejectedVendor.[`_id`] === sourcepointId,
```

Reply:
I’m afraid that’s not going to work with prettier 😞